actions/class.RestItems.php

```diff
@@ -21,15 +21,18 @@
  */
 class taoItems_actions_RestItems extends tao_actions_CommonRestModule
 {
-
-	public function __construct()
+    /**
+     * @return taoItems_models_classes_CrudItemsService
+     */
+    protected function getCrudService()
     {
-		parent::__construct();
-		//The service taht implements or inherits get/getAll/getRootClass ... for that particular type of resources
-		$this->service = taoItems_models_classes_CrudItemsService::singleton();
-	}
+        if (!$this->service) {
+            $this->service = taoItems_models_classes_CrudItemsService::singleton();
```

Comment:
Shouldn't we use `ServiceLocator`?

Reply:
Unfortunately no, it is not a configurableService ...